### ISSUE TITLE
SQSError 404 Still happening

### ISSUE THREAD

**Issue by andrewgross** [spulec/PyQS#24]
```
...
local/lib/python2.7/site-packages/boto/connection.py", line 1207, in get_status
    raise self.ResponseError(response.status, response.reason, body)
SQSError: SQSError: 404 Not Found
<?xml version="1.0"?><ErrorResponse xmlns="http://queue.amazonaws.com/doc/2012-11-05/"><Error><Type>Sender</Type><Code>ReceiptHandleIsInvalid</Code><Message>The receipt handle &quot;AQEBqF18AlKqQ2nbvbXRyDEznxoHXZgy2nb50toBH6XuNK/Oe2qoOKDF+E9Etye2VMd7gwbh/ZlPr9Ry4ZK1RJHJf+r5ZvqXPaC/IxKmfkRlowuaARPSHGLh95W/lNEmoD/uVmgWNe/oY+BbrzVCVV0S9LorhAImv6C/4sfYrZ0Yjkagt5pnk3VGw8PkChPm3P289vb9vhxag7TcTCNqaycEYaKj1yurx6kZvtI88yjso+TM7wltHfdnkXVw/Y1ZBqKYAqh7APBQHW4dobzKTAj25myltqxvgpxt2o2w8TYj2iubEI+tGRAaqA2/ghulnN8VQV1Vih4HlvCVxT/FsPopsg==&quot; is not valid for this queue.</Message><Detail/></Error><RequestId>6870d6ef-b7a9-5046-9195-98ad09f88919</RequestId></ErrorResponse>
```

I think the issue is arising from pickling the `Queue` object when adding it to the internal queue.  See https://github.com/boto/boto/blob/develop/boto/sqs/queue.py#L40-L46

The solution should be to only add the `queue.id` to the internal queue, and then make the call.  We can create a fake queue object, or just call `get_status` manually.

**Comment by andrewgross** [comment]
Trying a fix in 92adfa0

**Comment by andrewgross** [comment]
Seems to be gone, but very hard to reproduce properly. Leaving the ticket open.
